cscore/src/main/native/cpp/UsbCameraProperty.h

```diff
@@ -26,6 +26,15 @@ class UsbCameraProperty : public PropertyImpl {
   UsbCameraProperty() = default;
   explicit UsbCameraProperty(wpi::StringRef name_) : PropertyImpl{name_} {}
 
+  // Software property constructor
+  UsbCameraProperty(wpi::StringRef name_, unsigned id_, CS_PropertyKind kind_,
```

Comment:
You don't technically need the underscores here. `id{id}` initializes the `this` version of the variable properly and any use of `name`, for example, in the derived constructor will reference the local variable instead of the shadowed one in `this`. Having underscores does make it clearer to read though, so :man_shrugging:.